### PR TITLE
[dynamix] docs

### PR DESCRIPTION
**Pull request by aleksey-su**
Add documentation for cloud-provider-dynamix

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @aleksey-su. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @aleksey-su. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mortelumina98. I've started the workflow for label 'deploy/web/test'...

File: ee/candi/cloud-providers/dynamix/openapi/doc-ru-cluster_configuration.yaml [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+        description: |
+          IP CIDR, который будет назначен внутреннему сегменту этой ViNS.
+      nameservers:
+        description: |
+          Список IP-адресов DNS-серверов.
+      layout:
+        description: |
+          Название схемы размещения.
+
+          [Подробнее](https://deckhouse.io/products/kubernetes-platform/documentation/v1/modules/030-cloud-provider-dynamix/cr.html#layouts)  о возможных схемах размещения провайдера.
```
Comment (mortelumina98) [inline ee/candi/cloud-providers/dynamix/openapi/doc-ru-cluster_configuration.yaml:129]:
```suggestion
          [Подробнее](https://deckhouse.ru/products/kubernetes-platform/documentation/v1/modules/030-cloud-provider-dynamix/cr.html#layouts)  о возможных схемах размещения провайдера.
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mortelumina98. I've started the workflow for label 'deploy/web/test'...

File: ee/candi/cloud-providers/dynamix/openapi/doc-ru-cluster_configuration.yaml [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+        description: |
+          IP CIDR, который будет назначен внутреннему сегменту этой ViNS.
+      nameservers:
+        description: |
+          Список IP-адресов DNS-серверов.
+      layout:
+        description: |
+          Название схемы размещения.
+
+          [Подробнее](https://deckhouse.ru/products/kubernetes-platform/documentation/v1/modules/030-cloud-provider-dynamix/cr.html#layouts)  о возможных схемах размещения провайдера.
```
Comment (mortelumina98) [inline ee/candi/cloud-providers/dynamix/openapi/doc-ru-cluster_configuration.yaml:129]:
```suggestion
          [Подробнее](../../../../modules/030-cloud-provider-dynamix/layouts.html)  о возможных схемах размещения провайдера.
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mortelumina98. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mortelumina98. I've started the workflow for label 'deploy/web/test'...

File: ee/candi/cloud-providers/dynamix/openapi/doc-ru-cluster_configuration.yaml [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+        description: |
+          IP CIDR, который будет назначен внутреннему сегменту этой ViNS.
+      nameservers:
+        description: |
+          Список IP-адресов DNS-серверов.
+      layout:
+        description: |
+          Название схемы размещения.
+
+          [Подробнее](https://deckhouse.ru/products/kubernetes-platform/documentation/v1/modules/030-cloud-provider-dynamix/layouts.html)  о возможных схемах размещения провайдера.
```
Comment (mortelumina98) [inline ee/candi/cloud-providers/dynamix/openapi/doc-ru-cluster_configuration.yaml:129]:
```suggestion
          [Подробнее](layouts.html)  о возможных схемах размещения провайдера.
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mortelumina98. I've started the workflow for label 'deploy/web/test'...

File: ee/candi/cloud-providers/dynamix/openapi/doc-ru-cluster_configuration.yaml [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+                  x-doc-example: |
+                    ```yaml
+                    taints:
+                    - effect: NoExecute
+                      key: ship-class
+                      value: frigate
+                    ```
+            instanceClass:
+              description: |
+                Частичное содержимое полей [DynamixInstanceClass](https://deckhouse.ru/products/kubernetes-platform/documentation/v1/modules/030-cloud-provider-dynamix/cr.html#dynamixinstanceclass).
```
Comment (mortelumina98) [inline ee/candi/cloud-providers/dynamix/openapi/doc-ru-cluster_configuration.yaml:116]:
```suggestion
                Частичное содержимое полей [DynamixInstanceClass](cr.html#dynamixinstanceclass).
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @aleksey-su. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mortelumina98. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Pull request merged**